student.rb

```diff
@@ -0,0 +1,14 @@
+# create a student class 
+class student < person
```

Comment:
- [ ] Kindly capitalize the `s` in Student and `p` in Person as they are both class names 👍🏿